pkg/gui/controllers/custom_patch_options_menu_action.go

```diff
@@ -182,12 +183,23 @@ func (self *CustomPatchOptionsMenuAction) handlePullPatchIntoNewCommit() error {
 		return err
 	}
 
-	return self.c.WithWaitingStatus(self.c.Tr.RebasingStatus, func(gocui.Task) error {
-		commitIndex := self.getPatchCommitIndex()
-		self.c.LogAction(self.c.Tr.Actions.MovePatchIntoNewCommit)
-		err := self.c.Git().Patch.PullPatchIntoNewCommit(self.c.Model().Commits, commitIndex)
-		return self.c.Helpers().MergeAndRebase.CheckMergeOrRebase(err)
-	})
+	commitIndex := self.getPatchCommitIndex()
+	return self.c.Helpers().Commits.OpenCommitMessagePanel(
+		&helpers.OpenCommitMessagePanelOpts{
+			CommitIndex:     commitIndex,
+			InitialMessage:  "",
```

Comment:
In the original PR (#2436) this was initialized to `Split from "<original message>"`, which I don't think is useful. You always have to delete that text before typing your own.

I briefly considered initializing this to the original message, which would be useful in the case that you want to keep the message in the second commit, and then reword the first commit afterwards (which, for me, happens in roughly 50% of the cases). However, it would be inconvenient in the other 50%; and since you can get the message of the previous commit by pressing down and up arrow, it's probably best to start with an empty panel.

Reply:
Does pressing up take you straight to the original commits message or do you need to press up a few times to get there? Would be good if it could go straight there

Reply:
You had to press down and up (or up and down), but I pushed a fixup (fc5730ac51) that makes it so that you only need to press up once.